### ISSUE TITLE
Support for Android Themed app icon feature..

### ISSUE THREAD

**Issue by Domiiniik** [hrydgard/ppsspp#17988]
### What should happen

Make PPSSPP on Android support themed app icon API feature for consistency with other UI elements if user is using "app icons following color palette theming"
Example:
![Screenshot_20230827_211838_One UI Home](https://github.com/hrydgard/ppsspp/assets/30195961/489d297d-7194-4acf-8891-ca0b23f53aec)
More information on this API and its implementation here:
https://developer.android.com/develop/ui/views/launch/icon_design_adaptive

### Who would this benefit

Users who like consistency in system theming.

### Platform (if relevant)

Android

### Games this would be useful in

PPSSPP

### Other emulators or software with a similar feature

-

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by ANR2ME** [comment]
Themed icons usually for internal/system apps isn't?

**Comment by Domiiniik** [comment]
> Themed icons usually for internal/system apps isn't?
> 
> And since we have Gold version too, enforcing a gold foreground color on a system background color may looks weird on some background.

Not really, multiple apps support that that are third party, and there should not be a problem with colors either more info on Googles website about this as i mentioned up above.

**Comment by hrydgard** [comment]
We're already using adaptive icons for the shape. Didn't know adapting the color was also possible, but seems like it just makes icons even less distinct from each other... That said, I can see argument both ways.
